devops/base/server/deploy.yaml

```diff
@@ -123,6 +123,27 @@ spec:
           volumeMounts:
             - name: rocketchat-uploads
               mountPath: /app/uploads
+          env:
```

Comment:
Is the secret created by an operator or something? can we switch this to use vault instead? Just to be consistent with the other secrets.